doc/src/manual/strings.md

```diff
@@ -333,9 +333,35 @@ julia> string(greet, ", ", whom, ".\n")
 "Hello, world.\n"
 ```
 
-Constructing strings like this can become a bit cumbersome, however. To reduce the need for these
-verbose calls to [`string()`](@ref), Julia allows interpolation into string literals using `$`,
-as in Perl:
+Julia also provides `*` for string concatenation:
+
+```julia
```

Comment:
Probably worth starting to re-add `jldoctest`s when new examples are added; it can be done bit by bit for anything in `manual/`. Since this one uses variables from the above block you'll need to use named blocks, i.e.

    ```jldoctest my-name-here

to share a namespace between each code block.